src/Utils/VueJsFunctionScanner.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Gettext\Utils;
+
+class VueJsFunctionScanner extends JsFunctionsScanner
+{
+    /**
+     * Number of lines that the the script of the template starts
+     * @var int
+     */
+    public $lineOffset = 0;
+
+    /**
+     * @inheritdoc
+     */
+    public function getFunctions(array $constants = [])
+    {
+        $functions = parent::getFunctions($constants);
+
+        // Add line offset to the functions because vue templates contain template at the top and the script is below.
+        // When we parse, we parse only the script part so we need to add the line number
+        $functions = array_map(function ($v) {
+            $v[1] += $this->lineOffset - 1;
+            return $v;
+        }, $functions);
+
+        return $functions;
+    }
+}
```

Comment:
I think this class is not necessary, the lineOffset could be added as an option for the JsFunctionScanner, so it can be reused by other javascript extractors

Reply:
Agreed, I'll make the changes after my vacation (14th of May). Hope this can sit here until then.